alert/blueprints/application/routes.py

```diff
@@ -1,12 +1,10 @@
 from flask import Blueprint, render_template
-from flask_babel import _
 
 
 application = Blueprint("user", __name__)
 
 
 @application.route("/")
-@application.route("/home")
```

Comment:
When the application is deployed, the web server will route `/*` (all requests) to `/` as we only have the one page for now. This means we only need to specify the route for `/` , as `/home` will be redirected to `/`